pkg/rpcclient/wsclient_test.go

```diff
@@ -471,6 +472,22 @@ func TestWSFilteredSubscriptions(t *testing.T) {
 				require.NoError(t, json.Unmarshal(param.RawMessage, filt))
 				require.Nil(t, filt.Sender)
 				require.Equal(t, util.Uint160{0, 42}, *filt.Signer)
+				require.Nil(t, filt.Type)
+			},
+		},
+		{"transactions type",
+			func(t *testing.T, wsc *WSClient) {
+				typeTx := "removed"
+				_, err := wsc.ReceiveTransactions(&neorpc.TxFilter{Type: &typeTx}, make(chan *transaction.Transaction))
+				require.NoError(t, err)
+			},
+			func(t *testing.T, p *params.Params) {
+				param := p.Value(1)
+				filt := new(neorpc.TxFilter)
+				require.NoError(t, json.Unmarshal(param.RawMessage, filt))
+				require.Nil(t, filt.Sender)
+				require.Nil(t, filt.Signer)
+				require.Equal(t, "removed", *filt.Type)
```

Comment:
We need a test for notary requests. The same idea, but with `ReceiveNotaryRequests`